presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeleteDeltaPageSource.java

```diff
@@ -104,14 +102,15 @@ public OrcDeleteDeltaPageSource(
             Map<String, OrcColumn> acidColumns = uniqueIndex(
                     reader.getRootColumn().getNestedColumns(),
                     orcColumn -> orcColumn.getColumnName().toLowerCase(ENGLISH));
-            List<OrcColumn> rowIdColumns = ImmutableList.of(
+
+            List<OrcColumn> rowIdColumns;
+            rowIdColumns = ImmutableList.of(
```

Comment:
Please take a look at https://github.com/prestosql/presto/pull/2930/files
-- this is undesirable change here:

```suggestion
            List<OrcColumn> rowIdColumns = ImmutableList.of(
```

Reply:
reverted.